tests/api/admin/controller/test_individual_admins.py

```diff
@@ -676,9 +691,12 @@ def test_individual_admins_post_create_on_setup(self):
                 AdminNotAuthorized,
                 self.manager.admin_individual_admin_settings_controller.process_post,
             )
-            self._db.rollback()
 
-        # The password is required.
+    def test_individual_admins_post_create_requires_password(self):
```

Comment:
This tests scenario 1, setting up the first admin, but not scenario 2, creating another admin with or without a password.

Reply:
Oops, yes, I mangled one of the tests. Fixing... :hourglass: